src/Gui/CommandDoc.cpp

```diff
@@ -1143,7 +1143,7 @@ void StdCmdDelete::activated(int iMsg)
                     //message for linked items
                     if (!affectedLabels.empty()) {
                         bodyMessageStream << qApp->translate("Std_Delete",
-                                                             "These items are linked to items selected for deletion and might break.\n\n");
+                                                             "These items are linked to items selected for deletion and might break.") << '\n\n';

```

Comment:
Not sure I did this correctly. I'm trying to eliminate superfluous whitespace from strings that are translated to avoid translators making mistakes.

Reply:
you must use double quotes in c++, afaik

Reply:
So all I'm missing is `<< "\n\n";` here ?